PendingReleaseNotes

```diff
@@ -1,3 +1,14 @@
+* RGW: RGW specific openssl algorithm.
+  * Removed openssl_engine_opts, no longer supports passing OpenSSL engine configurations in string format.
+  * Added openssl_conf configuration option to loading specified providers as the default providers
+    before other providers for RGWs.
+    The providers configuration must now be passed via the configuration file specified by openssl_conf.
+    Configuration file syntax follows the official OpenSSL standard, see
+    https://github.com/openssl/openssl/blob/master/doc/man5/config.pod
+    If specific provider is already loaded via openssl_conf and the default provider is still required,
+    it must be explicitly loaded either in the configuration file or in the code, see
+    https://github.com/openssl/openssl/blob/master/README-PROVIDERS.md
```

Comment:
i reworded the changelog for better readability:

```
* RGW: OpenSSL engine support deprecated in favor of provider support.
  - Removed `openssl_engine_opts` configuration option. OpenSSL engine configurations in string format are no longer supported.
  - Added `openssl_conf` configuration option for loading specified providers as default providers. Configuration file syntax follows the OpenSSL standard (see https://github.com/openssl/openssl/blob/master/doc/man5/config.pod). If the default provider is still required when using custom providers, it must be explicitly loaded in the configuration file or code (see https://github.com/openssl/openssl/blob/master/README-PROVIDERS.md).
```

could you please consider incorporating it?